skyportal/tests/utils/test_moving_objects.py

```diff
@@ -0,0 +1,237 @@
+import arrow
+import astropy.units as u
+import numpy as np
+import pandas as pd
+
+from skyportal.models import DBSession
+from skyportal.tests.external.test_moving_objects import add_telescope_and_instrument
+from skyportal.utils.calculations import radec_to_healpix
+from skyportal.utils.moving_objects import *
+
+
+def test_find_jplhorizon_obj():
```

Comment:
Any concerns about this needing its own place depending on an external service?

Reply:
The external folder is also green on failure like flaky (not for all the tests, so I'd rather keep this here and move it to external only if the JPL API happens to be flaky